utils/merge_histograms.py

```diff
@@ -30,33 +33,34 @@ def main():
   hist_path = "histograms_muonSFs_muonTriggerSFs_pileupSFs_bTaggingSFs"
   
   sample_paths = (
-    # Backgrounds
-    "backgrounds2018/TTToSemiLeptonic",
-    "backgrounds2018/TTToHadronic",
-    "backgrounds2018/TTTo2L2Nu",
+    # # Backgrounds
+    # "backgrounds2018/TTToSemiLeptonic",
+    # "backgrounds2018/TTToHadronic",
+    # "backgrounds2018/TTTo2L2Nu",
     
-    "backgrounds2018/ST_tW_antitop",
-    "backgrounds2018/ST_tW_top",
-    "backgrounds2018/ST_t-channel_antitop",
-    "backgrounds2018/ST_t-channel_top",
+    # "backgrounds2018/ST_tW_antitop",
+    # "backgrounds2018/ST_tW_top",
+    # "backgrounds2018/ST_t-channel_antitop",
+    # # # "backgrounds2018/ST_t-channel_top",
     
-    "backgrounds2018/DYJetsToMuMu_M-50",
-    "backgrounds2018/DYJetsToMuMu_M-10to50",
+    # "backgrounds2018/DYJetsToMuMu_M-50",
+    # "backgrounds2018/DYJetsToMuMu_M-10to50",
     
-    "backgrounds2018/WJetsToLNu",
+    # "backgrounds2018/WJetsToLNu",
     
-    "backgrounds2018/TTZToLLNuNu",
-    "backgrounds2018/TTZToLLNuNu_M-1to10",
+    # # # "backgrounds2018/TTZToLLNuNu",
+    # "backgrounds2018/TTZToLLNuNu_M-10",
+    # "backgrounds2018/TTZToLL_M-1to10",
     
-    "backgrounds2018/TTWJetsToLNu",
+    # "backgrounds2018/TTWJetsToLNu",
     
-    "backgrounds2018/ttHTobb",
-    "backgrounds2018/ttHToNonbb",
-    "backgrounds2018/ttHToMuMu",
+    # "backgrounds2018/ttHTobb",
+    # "backgrounds2018/ttHToNonbb",
+    # "backgrounds2018/ttHToMuMu",
     
-    "backgrounds2018/TTZZ",
-    "backgrounds2018/TTZH",
-    "backgrounds2018/TTTT"
+    # "backgrounds2018/TTZZ",
+    # "backgrounds2018/TTZH",
+    # # # "backgrounds2018/TTTT"
     
```

Comment:
Shouldn't we include these paths from the new file you created?

Reply:
Oh yeah of course! didn't think about that